solarwinds_apm/apm_config.py

```diff
@@ -530,7 +572,6 @@ def _snake_to_camel_case(key):
             self.agent_enabled = cnf_agent_enabled
 
         available_cnf = set(self.__config.keys())
-        # TODO after alpha: is_lambda
```

Comment:
As per Slack chat starting [here](https://swicloud.slack.com/archives/C05R52Y2GHK/p1698706088610519), we won't configure `token_*` in APM Python like we did in AO. So no longer needed.